lib/nightmare.js

```diff
@@ -80,8 +80,9 @@ function Nightmare(options) {
 
   process.setMaxListeners(Infinity);
   process.on('uncaughtException', function(err) {
-    console.error(err.stack);
+    debug(err.stack);
```

Comment:
Nice suggestion, feels a little hacky but I can't see much to go wrong as either the user exits in a handler of their own or the script exits on the next tick. Added and also added a comment about why it's being done this way.